lib/model.d.ts

```diff
@@ -325,7 +325,7 @@ export interface IncludeOptions {
   /**
    * Load further nested related models
    */
-  include?: Includeable[];
+  include?: Includeable[] | IncludeAll[];
```

Comment:
Unless I am missing something, the only valid option is `[{ all: true }]`, the array cannot contain other members, and `{ all: false }` is not valid.
I would also like to avoid duplicating this expression everywhere, it was like this in the past and resulted in inconsistent options. I would define
```ts
type IncludeOptions = Includable[] | [{ all: true }];

...

include?: IncludeOptions;
```